demo/mobile/src/app/config/config.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
For Keycloak config as we agreed above. Example is in `demo/mobile/src/app/config/keycloakExample.json` and `config.json` is empty as keycloak should not be run by default in the demo app. We only use keycloak once the keycloakConfig is defined in this file. This will be part of the docs.